pkg/liqonet/iptables/iptables.go

```diff
@@ -736,7 +738,8 @@ func getChainRulesPerCluster(tep *netv1alpha1.TunnelEndpoint) (map[string][]IPTa
 	chainRules[liqonetInputChain] = append(chainRules[liqonetInputChain],
 		IPTableRule{"-d", remotePodCIDR, "-j", getClusterInputChain(clusterID)})
 	chainRules[liqonetForwardingChain] = append(chainRules[liqonetForwardingChain],
-		IPTableRule{"-d", remotePodCIDR, "-j", getClusterForwardChain(clusterID)})
+		IPTableRule{"-d", remotePodCIDR, "-j", getClusterForwardChain(clusterID)},
+		IPTableRule{"-i", consts.DeviceName, "-d", localRemappedExternalCIDR, "-j", DROP})
```

Comment:
Would add an iptables comment, clarifying the purpose of the rule, and which cluster it refers to.